src/libopensc/card-openpgp.c

```diff
@@ -510,20 +545,19 @@ pgp_get_card_features(sc_card_t *card)
 	size_t i;
 	pgp_blob_t *blob, *blob6e, *blob73;
 
+	/* parse card capabilities from historical bytes in ATR */
 	if (hist_bytes_len > 0) {
```

Comment:
This part deals with the hist_bytes in the ATR.
They are optional, but if present, they serve as good fallback values.